### PR TITLE
Login

### PR DESCRIPTION
**Pull request by Maxim-Gumeniuk**

### CODE REVIEW

**Comment by Maxim-Gumeniuk** [comment]
[sergienko339@gmail.com](mailto:sergienko339@gmail.com)

File: src/server.ts [hunk 1]
```diff
@@ -24,7 +25,12 @@ const start = async (): Promise<void> => {
         console.log(`Server started on ${PORT}`); 
     });
 
-    connectUsersDb()
+    await connectUsersDb()
```
Comment (SpaceHolderdJs) [inline src/server.ts]: It`s fine to left it separate

File: src/helpers/normalize-user.ts [hunk 2]
```diff
@@ -0,0 +1,4 @@
+export const normalizeUser = ({ _id, email }: any) => ({
```
Comment (SpaceHolderdJs) [inline src/helpers/normalize-user.ts:1]: `Partial<UserInterface>`

File: src/services/login/index.ts [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import { userModel } from "@/db/users"
+
+const findUserByEmail = ( email: string) => {
+    const user = userModel.findOne({ email })
+
+    console.log(user);
```
Comment (Maxim-Gumeniuk) [inline src/services/login/index.ts]: delete

File: src/routes/main/index.ts [hunk 4]
```diff
@@ -0,0 +1,10 @@
+// import { loginController } from "@/controllers/auth/login";
```
Comment (Maxim-Gumeniuk) [inline src/routes/main/index.ts]: d

**Pull request merged**